src/redisearch_rs/inverted_index/src/lib.rs

```diff
@@ -252,14 +253,16 @@ pub struct IndexBlock {
     buffer: Vec<u8>,
 }
 
+static TOTAL_BLOCKS: AtomicUsize = AtomicUsize::new(0);
+
 impl IndexBlock {
     const SIZE: usize = std::mem::size_of::<Self>();
 
     /// Make a new index block with primed with the initial doc ID. The next entry written into
     /// the block should be for this doc ID else the block will contain incoherent data.
     ///
     /// This returns the block and how much memory grew by.
-    pub fn new(doc_id: t_docId) -> (Self, usize) {
+    fn new(doc_id: t_docId) -> (Self, usize) {
```

Comment:
What is reason behind the  change the visibility?

Reply:
I realized nothing externally makes a new index block (and nothing should really). So just updated it as a quality of life thing.